doc/source/examples/incompressible-flow/2d-lid‐driven-cavity-flow/lid‐driven-cavity-flow.rst

```diff
@@ -0,0 +1,306 @@
+
+.. image:: images/lethe_ghia_re_400_comparison.png
+    :alt: re_400_comparison
+    :align: center
+
+We note that the agreement is perfect. This is not surprising, especially considering that these results were obtained at relatively low Reynolds number.
+
+.. note:: 
+    The vtu files generated by Lethe are compressed archives. Consequently, they cannot be postprocessed directly. Although they can be easily post-processed using Paraview, it is sometimes necessary to be able to work with the raw data. The python library pyvista allows us to do this.
+
+
+
+Case at Re=7500
+---------------
+
+We now consider the case at a Reynolds number of 7500. At this value of the Reynolds number, the ``steady`` solver will generally not converge as the problem is too non-linear (or too stiff). A workaround for this issue is to use an adjoint time-stepping strategy. This strategy consists in transforming the steady-state problem into a transient problem and to use an increasingly large time-step to reach a steady-state solution. This method is called ``steady_bdf`` in Lethe.
+
+It can be used by modifying the ``simulation control`` subsection:
+
+.. code-block:: text
+
+  subsection simulation control
+    set method                  = steady_bdf
+
+    set adapt = true
+    # Maximum CFL value
+    set max cfl                      = 1000
+    # Tolerance at which the simulation is stopped
+    set stop tolerance               = 1e-10
+  
+    # Adaptative time step scaling
+    set adaptative time step scaling = 1.05
+
+    set time step = 0.001
+  end
+
+The ``adapt`` parameter allows dynamic time-step adaptation. This feature is also used in transient simulations to carry simulations at a constant CFL number. The ``max cfl`` controls the maximum value of the CFL reached during the simulation. Remember that Lethe is an implicit solver and, as such, can theoretically manage absurdly large values of the CFL. The ``stop-tolerance`` controls the initial tolerance of a time-step below which steady-state will be considered reached.  The ``time step`` controls the initial value of the time step and, finally, the ``adaptative time step scaling`` controls the rate of increase of the time step. The increase of the time step follows:
+
+.. math::
+
+  \Delta t_{n+1} = \alpha \Delta t_{n}
+
+
+where :math:`\alpha` is the ``adaptative time step scaling`` .
+
+A complimentary strategy is to also start the problem from a closer initial condition. This is achieved with the ``initial conditions`` subsection:
+
+.. code-block:: text
+
+  subsection initial conditions
+    set type      = viscous
+    # viscosity for viscous initial conditions
+    set viscosity = 0.1
+  end
+
+Here we use ``viscous`` initial condition in which an auxiliary steady-state problem at a higher viscosity is used to generate a more realistic initial condition to the problem at hand.
+
+Results
```

Comment:
I would delete this and keep only one subsection for the higher Reynolds case

Reply:
You are right, it detracts the focus a bit. I'll remove that part. We can leave it to another case.